### ISSUE TITLE
Trouble in following the documentation order

### ISSUE THREAD

**Issue by kevinjhanna** [mercadolibre/api#30]
This is an issue I ran into while looking how to connect via oAuth following the steps from the [documentation](http://developers.mercadolibre.com/authentication-and-authorization/)

Extract from the "Server-side flow" documentation:

>In this case, **the first steps of the flow are the same**. However, when the user grants the permissions to your application he will be redirected to the url as specified when you registered your application with an access_code parameter.

>http://your_url?access_code=...

So then I read the first steps from the "client-side flow"

> You don't need to pass your redirect URL. Just make a GET request to this URL:
>https://auth.mercadolibre.com.ar/authorization?response_type=token...

Minutes later, when I realized this wasn't working, I kept on reading the explanation of how the server-side flow works:

>If a refresh token is present in the authorization code exchange, then it may be used to obtain new access tokens at any time. This type of access is called **offline**, since the user does not have to be present at the browser when the application obtains a new access token.

So I thought, "hmm, this is for offline access without the browser, it's not what I am looking for..."

But then I decided to try it anyway and, of course, it worked.
>https://auth.mercadolibre.com.ar/authorization?response_type=code...

## tl;dr:
The documentation is great except for that little part in the "server-side flow":
>the first steps of the flow are the same.

I would suggest removing that phrase from the server-side flow and changing the example url
>http://your_url?access_code=...

to

>http://your_url?code=...

As it is shown in the "server-side flow explained" section.

That's all, thanks.
